src/components/core/Alert.vue

```diff
@@ -19,12 +19,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   <div v-if="alert">
     <v-alert
       :value="true"
-      :color="alert.getColor()"
+      :type="alert.getColor()"
       :icon="alert.getIcon()"
-      class="ma-0"
+      :class="getColor(alert.getColor())"
```

Comment:
Are these colour classes are built into Vuetify?

Reply:
Yup, the material color palette https://v15.vuetifyjs.com/en/framework/colors

Reply:
Matches the Cylc logo pretty well!